app/src/main/java/com/example/androidacademy/movieslist/MovieAdapter.kt

```diff
@@ -59,17 +60,17 @@ class MovieViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView){
                 .apply(imageOption)
                 .into(poster)
 
-        ratings.rating = movie.ratings/2
+        ratings.rating = movie.ratings / 2
         movieName.text = movie.title
         pgName.text = movie.minimumAge.toString()
-        reviewTV.text = ""+movie.reviews+" MIN"
+        reviewTV.text = "" + movie.reviews + " MIN"
         genres.text = movie.genres.joinToString(", ") { it.name }
-        runtime.text =" "+movie.runtime+" MIN"
+        runtime.text = " " + movie.runtime + " MIN"
```

Comment:
Зачем здесь используется " "? Похоже, что отсутствует отступ в разметке.
Можно использовать макроподстанивки в строках
```suggestion
        runtime.text = " ${movie.runtime} MIN"
```